jobs/JEVS_GLOBAL_ENS_WAVE_GRID2OBS_STATS

```diff
@@ -60,7 +60,7 @@ export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${MODELNAME}/${gefs_ve
 export ARCgefs=${COMIN}/prep/${COMPONENT}
 export COMINgdasnc=${COMIN}/prep/${COMPONENT}
 
-export COMOUT=${COMOUT:-${COMIN}/${STEP}/${COMPONENT}}
+export COMOUT=${COMOUT:-$(compath.py ${NET}/${evs_ver})/${STEP}/${COMPONENT}}
```

Comment:
As noted by Alicia, she beat me to the punch, this needs `-o` added. I didn't get a chance to explain it in our little lesson earlier today. But please change line 63 to: 
```
export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/${STEP}/${COMPONENT}}
```
I can explain why in chat tomorrow.